src/components/ListItem/ListItem.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+const ListItem = ({ name }) => {
+  return <StyledListItem role="listitem">{name}</StyledListItem>
```

Comment:
Change the HTML tag to `li` and you don't necessarily need the role here.

Reply:
Fixed